core/java/iesi-rest-without-microservices/src/main/java/io/metadew/iesi/server/rest/script/dto/ScriptDtoRepository.java

```diff
@@ -196,6 +200,22 @@ private ScriptExecutionDto mapScriptExecutionDto(CachedRowSet cachedRowSet) thro
         );
     }
 
+    /**
+     * This method sort the given List according to the provided sort
+     * @param pageable - The pageable containing the Sort object - can be null
+     * @param scriptDtoList - The list to sort
+     */
+    private void resultSorting(Pageable pageable, List<ScriptDto> scriptDtoList) {
+        if (pageable == null || pageable.getSort().isUnsorted())
+            return;
+        pageable.getSort().forEach(order -> {
+            // add other sorting here
+            if (order.getProperty().equalsIgnoreCase("NAME")) {
+                scriptDtoList.sort((Comparator.comparing(ScriptDto::getName)));
+            }
+        });
+    }
+
```

Comment:
A more elagant way is to create a LinkedHashMap in the getAll() and getByName() function. Tests succeed if this change is made